libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/NavigationPerformanceMetadata.kt

```diff
@@ -0,0 +1,57 @@
+package com.mapbox.services.android.navigation.v5.internal.navigation
+
+internal data class NavigationPerformanceMetadata(
+    val version: String,
+    val screenSize: String,
+    val country: String,
+    val device: String,
+    val abi: String,
+    val brand: String,
+    val ram: String,
+    val os: String,
+    val gpu: String,
+    val manufacturer: String
+) {
+
+    companion object {
+        @JvmStatic
+        internal fun builder() = Builder()
```

Comment:
looks like this `Metadata` is only for internal usage? may be better move class into `internal` package and use default values instead `Builder`

Reply:
Added Fixme for class to mark it should be refactored as you described above when NavigationTelemetry and NavigationMetricsWrapper are migrated to Kotlin